app/models/ChannelType.scala

```diff
@@ -18,13 +18,18 @@ package models
 
 sealed trait ChannelType
 
+sealed abstract class WithName(name: String) {
+  override def toString: String = name
+}
+
 object ChannelType extends Enumerable.Implicits {
-  case object web extends ChannelType
-  case object api extends ChannelType
+  case object Web     extends WithName("web") with ChannelType
+  case object Api     extends WithName("api") with ChannelType
+  case object Deleted extends WithName("deleted") with ChannelType
 
-  val values: Seq[ChannelType] = Seq(web, api)
+  val values: Seq[ChannelType] = Seq(Web, Api)
 
   implicit val enumerable: Enumerable[ChannelType] =
-    Enumerable(values.map(v => v.toString -> v): _*)
+    Enumerable(values.map(v => v.toString.toLowerCase() -> v): _*)
```

Comment:
We don't need this change because of the WithName above

```suggestion
    Enumerable(values.map(v => v.toString -> v): _*)
```